app/modules/pagination/actions/pagination-fetch-all.js

```diff
@@ -0,0 +1,25 @@
+import {fetchPage} from "./pagination-fetch-page"
+import {paginationSetFetching} from "./pagination-set-fetching"
+import {paginationSetAssignmentURL} from "./pagination-set-assignment-url"
+import {nextPage} from "../selectors"
+
+// PUBLIC: Async thunk action for fetching all pages of repos
+export const fetchAllPages = (assignmentURL) => {
+  return (dispatch, getState) => {
+    var urlObj = new URL(assignmentURL)
+    var repoURL = `${urlObj.origin}/api/internal/${urlObj.pathname}/repos`
+    dispatch(paginationSetAssignmentURL(assignmentURL))
```

Comment:
Set pagination assignment URL so we can keep track if the user changes the URL and pagination information is out of date